icons/thread.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="none" viewBox="0 0 18 18"><path fill="currentColor" d="M5 5.25a.75.75 0 0 0 0 1.5h8a.75.75 0 0 0 0-1.5H5ZM5 8.25a.75.75 0 0 0 0 1.5h4a.75.75 0 1 0 0-1.5H5Z"/><path fill="currentColor" fill-rule="evenodd" d="M3 .25A2.75 2.75 0 0 0 .25 3v14a.75.75 0 0 0 1.2.6L4.916 15c.217-.162.48-.25.75-.25H15A2.75 2.75 0 0 0 17.75 12V3A2.75 2.75 0 0 0 15 .25H3ZM1.75 3c0-.69.56-1.25 1.25-1.25h12c.69 0 1.25.56 1.25 1.25v9c0 .69-.56 1.25-1.25 1.25H5.666a2.75 2.75 0 0 0-1.65.55L1.75 15.5V3Z" clip-rule="evenodd"/></svg>
```

Comment:
This icon includes the padding from the compound design. There are some issues with that: Design wants to have a 1px padding for all icons by default. If we scale for instance the 18px icon to 12px or 24px we will have a slightly different padding. In some other cases design used a completely different or no padding for the icons in some places.

In my opinion the icon SVGs should not contain any padding. We can add the paddings by adding CSS rules as required.

Reply:
Discussed this with @nadonomy the plan is to make sure ALL icons created are exported and have the same viewbox.
Some icons will be adjusted so that they visually look of a similar size as the human eye sometimes perceive different shape differently.

At that point having a 1px padding does not impact anyone as it would be consistent across all icons.

What is important to note is that no developer should ever have to write `padding-top: -1px` to manually adjust the vertical alignment of an icon with the text baseline. This should come for free. 
Anything that falls short of that should be logged as an issue and we should re-generate the source icon